src/OrbitQt/DoubleClickableLabelTest.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QTest>
+
+#include "DoubleClickableLabel.h"
+
+namespace orbit_qt {
+
+TEST(DoubleClickableLabel, DoubleClickedSignalledOnDoubleClick) {
+  DoubleClickableLabel label;
+  bool double_clicked_signalled = false;
+  QObject::connect(&label, &DoubleClickableLabel::DoubleClicked,
+                   [&double_clicked_signalled] { double_clicked_signalled = true; });
```

Comment:
Nit: For testing signals there is `QSignalSpy` (https://doc.qt.io/qt-5/qsignalspy.html). But the current solution is also fine with me.

Reply:
It seems to work well for this case, done.